src/app/loaders/express/locals.ts

```diff
@@ -47,6 +49,8 @@ const environment = ejs.render(
     var spcpCookieDomain = "<%= spcpCookieDomain%>"
     // Old SPCP Cookie
     var oldSpcpCookieDomain = "<%= oldSpcpCookieDomain%>"
+    // Sms verification limit
+    var smsVerificationLimit = "<%= smsVerificationLimit%>"
```

Comment:
Looking at this again, I don't think we should send this env var to the client and ahve the client use it. We should use the `/count/free` endpoint to transmit this information. Reasoning is expanded on in the review...

With that said, we should update the endpoint to return an object instead of a number
```jsonc
// GET /count/free
{
  remainingFree: // number of free smses remaining?
  quota: // sms verification limit env var, but retrieved from the server side and transmitted to client instead
}
```

Reply:
done [here](https://github.com/opengovsg/FormSG/pull/2500/commits/f41056aaa14081db5463d8878a0161626d7d37a6)